src/main/java/seedu/duke/template/LcTemplate.java

```diff
@@ -112,13 +114,12 @@ public String toString() {
      * @throws DukeException If input String does not match a component.
      */
     @Override
-    public LoadComponent getComponent(String component) throws DukeException {
+    public LoadComponent getComponent(String component) {
+        assert component.equals("l") || component.equals("c");
```

Comment:
Same with this statement. I think avoid asserting in this function (even in the other classes) since it is kind of already taken care of by exception handling. The program will terminate in case the user enters an incorrect component

Reply:
Sure, will remove it.